test/github.ts

```diff
@@ -434,7 +429,9 @@ describe('GitHub', () => {
         commit => {
           // this commit is the 2nd most recent
           return commit.sha === 'b29149f890e6f76ee31ed128585744d4c598924c';
-        }
+        },
+        Number.MAX_SAFE_INTEGER,
+        true
```

Comment:
nit: this method takes enough arguments it might worth considering an interface.

Reply:
I agree, it's a code smell to have to provide the same default value. I'll follow up with a refactor PR.